tests/format/flow/function-parentheses/jsfmt.spec.js

```diff
@@ -1,3 +1,6 @@
-run_spec(__dirname, ["flow", "babel"]);
-run_spec(__dirname, ["flow", "babel"], { trailingComma: "all" });
-run_spec(__dirname, ["flow", "babel"], { arrowParens: "avoid" });
+// [prettierx] test with babel-flow
+// [TBD] skip TypeScript for now
+// (known parse error with babel-ts parser)
```

Comment:
@brodybits TODO: raise an issue to track this one

Reply:
- [ ] These comments are not quite right. I will push an update (again) to fix this.